### PR TITLE
chore: clean up README

### PR DESCRIPTION
**Pull request by Swizbiz**
https://github.com/artipie/artipie/issues/1031

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
 
 Artipie HTTP client implementation.
 
+If you have any question or suggestions, do not hesitate to create an issue or contact us in
+[Telegram](https://t.me/artipie).  
+Artipie [roadmap](https://github.com/orgs/artipie/projects/3).
+
 To install add this dependency to `pom.xml` file:
 ```xml
 <dependency>
```
Comment (olenagerasimova) [inline README.md:22]: I'd suggest to add link to https://github.com/artipie/http for words "HTTP module" in the sentence right after maven dep xml

Comment (olenagerasimova) [inline README.md:22]: > Client module **IS** targeted to implement HTTP client on top of these abstractions

"is" is missing in this sentence

Comment (olenagerasimova) [inline README.md:22]: > `ClientSlices` is **AN** entry point for usage of this module:

"an" is missing in this sentence

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Swizbiz thanks, looks good, although I have comments, please, check

**Comment by Swizbiz** [comment]
> @Swizbiz thanks, looks good, although I have comments, please, check

done

**Review by dgarus — DISMISSED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -15,10 +26,10 @@ To install add this dependency to `pom.xml` file:
 </dependency>
 ```
 
-Artipie HTTP module provides HTTP abstractions. Client module targeted to implement
+Artipie [HTTP module](https://github.com/artipie/http) provides HTTP abstractions. Client module is targeted to implement
 HTTP client on top of these abstractions.
 
-- `ClientSlice` - is a `Slice` that sends request to remote TCP endpoint and returns
+- `ClientSlice` - is an `Slice` that sends request to remote TCP endpoint and returns
```
Comment (olenagerasimova) [inline README.md:32]: why "an"? Slice starts with consonant... "A" is transformed into "an" only if the word starts with vowel: an entry, an opportunity, but a pen, a slice, a connection...

**Review by olenagerasimova — APPROVED** [review]
@Swizbiz thanks

**Pull request merged**